guides/common/modules/proc_configuring-repositories.adoc

```diff
@@ -25,40 +25,47 @@ ifdef::foreman-el,katello[]
 include::proc_configuring-repositories-el.adoc[]
 
 +
-. Enable Ruby 2.7 module:
+. If the Ruby 2.5 module has already been enabled, you need to perform a module reset.
```

Comment:
How does the user know if it is already enabled? (`dnf module list --enabled`).

Reply:
So which order should we take?

A) Try to enable and describe steps if it fails:

* Enable foreman/katello module
* If that fails, check `dnf module list --enabled`
* Reset ruby and/or postgresql
* Try again

B) prevent failure and instead be more explicit:

* Check `dnf module list --enabled`
* Reset ruby and/or postgresql if enabled
* Enable foreman/katello module

Reply:
Given this is the installation documentation (not upgrade), I'd go with "enable" with a "note: if this errors our because you already have blah enabled, you need to reset via `...`")